daily-challenges/2024-10/15/khanh5923_15.spec.js

```
@@ -0,0 +1,46 @@
+  else console.log("Tôi không đủ tiền mua");
+}
+test("daily/10/15_multiple-pages", async ({context}) => {
+  const page_TGDD = await context.newPage();
+  const page_DDV = await context.newPage();
+
+  // TGDD
+  await page_TGDD.goto(`https://www.thegioididong.com/`);
+  await page_TGDD.click(`//div[@class="icon-close-popup"]`);
+  await page_TGDD.locator(`//input[@id='skw']`).fill(`iPhone 16`);
+  await page_TGDD.click(`//button[@aria-label='button suggest search']`);
+  let priceTGDD = await page_TGDD
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'priceTGDD' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)